roxie/ccd/ccdserver.cpp

```diff
@@ -11965,7 +11965,7 @@ class CRoxieServerDiskWriteActivityFactory : public CRoxieServerMultiOutputFacto
         Owned<IHThorDiskWriteArg> helper = (IHThorDiskWriteArg *) helperFactory();
         isTemp = (helper->getFlags() & TDXtemporary) != 0;
         setNumOutputs(helper->getTempUsageCount());
-        if (_kind!=TAKdiskwrite)
+        if (_kind != TAKspillwrite)
```

Comment:
Should this also check (_kind!=TAKdiskwrite)?

Reply:
no - as far as I know numOutputs should only be non 0 if it is a spill